cluster-up/cluster/kind/check-cluster-up.sh

```diff
@@ -69,6 +69,11 @@ export CRI_BIN=${CRI_BIN:-$(detect_cri)}
     fi
     ${kubectl} wait -n kubevirt kv kubevirt --for condition=Available --timeout 15m
 
+    if [[ "$KUBEVIRT_PROVIDER" =~ "sriov" ]]; then
+      # Some SR-IOV tests require Kubevirt CPUManager feature
+      ${kubectl} patch kubevirts -n kubevirt kubevirt --type=json -p='[{"op": "replace", "path": "/spec/configuration/developerConfiguration/featureGates","value": ["CPUManager"]}]'
```

Comment:
Please consider adding a feature gate to the end of the existing list, instead of replacing the whole list, as it will enable future expansion.

Reply:
In case additional FG should be enabled I think there should be single a single patch call with all necessary FGs.
I didnt exported the FG name to var because its the only one at the moment.

Reply:
IMO this should not come as a hard dependency of the SR-IOV provider.
Do you see a problem with directly marking the need to have `CPUManager` as input from the caller?

Also, the patching is odd to me too.
- Why do you use `replace` and not a simple `add`?
- Why do you think it is better to assume there is only one FG? It will just make it harder for the next contributor to add other FGs in general.

Reply:
I dont see problem with setting CPUManager feature to always on, in fact in kubevirt/kubevirt its always on.
We can go with that. Let me know what you think.

Regarding the patch, I used "replace" because "add" didn't work for me in away I can add FG with one-liner.
When Kubevirt CR `developerConfiguration.featureGates` is not initialized, it will require patch for initializing it and another one for adding the FG.
Using "replace" the way  I did enable having one-liner.

Passing the provider name is the most simple way I could find to solve it and get the lane gating as soon as possible.
We can introduce some additional env var to hold a FG list, on a follow up PR in case it will be needed (I tried to keep things simple).

Reply:
@ormergi I am not convinced that here is the right place to patch kubevirt? IMHO it should go into github.com/kubevirt/kubevirt/tree/main/hack/cluster-deploy.sh. WDYT?

Reply:
@dhiller WDYT?

Reply:
This looks good to me as these tests are not failing in kubevirt - we just need to ensure that we have the cluster correctly configured in the kubevirtci presubmit which is what this script is used for so I think it is in the right place. 

Can we move this forward to get the sriov kubevirtci presubmit back to being required?

Reply:
> @dhiller Changing deploy-cluster.sh affect cluster-sync flow, I am not sure its we should do that. Please note "check-up-kind-sriov" calls "kind/check-cluster-up.sh" directly, and "kind/check-cluster-up.sh" deploy kubevirt (from nightly release yamls).

@ormergi makes sense, thanks for the explanation!